packages/firestore/src/local/memory_remote_document_cache.ts

```diff
@@ -16,25 +16,42 @@
 
 import { Query } from '../core/query';
 import {
+  documentKeySet,
   DocumentMap,
   documentMap,
   maybeDocumentMap
 } from '../model/collections';
-import { Document, MaybeDocument } from '../model/document';
+import { Document, MaybeDocument, NoDocument } from '../model/document';
 import { DocumentKey } from '../model/document_key';
 
 import { PersistenceTransaction } from './persistence';
 import { PersistencePromise } from './persistence_promise';
 import { RemoteDocumentCache } from './remote_document_cache';
+import { SnapshotVersion } from '../core/snapshot_version';
 
 export class MemoryRemoteDocumentCache implements RemoteDocumentCache {
   private docs = maybeDocumentMap();
+  private nextDocumentChanges = documentKeySet();
 
-  addEntry(
+  start(transaction: PersistenceTransaction): PersistencePromise<void> {
+    // Technically a no-op but we clear the set of changed documents to mimic
+    // the behavior of the IndexedDb counterpart.
+    //
+    // This is for testing only.
```

Comment:
What is for testing?  The clearing?  Or the entire "nextDocumentChanges" collection?  Please be more explicit or remove.

Reply:
Well, all of it I guess... But in this case I meant the clearing since start() should not have any data to clear. I removed that part of the comment.